namespace-resources/variables.tf

```diff
@@ -0,0 +1,11 @@
+variable "cluster" {
+    description = "What cluster are you deploying your namespace. I.E cloud-platform-test-1 "
+}
+
+variable "namespace" {
+    description = "Namespace you would like to create on cluster <application>-<environment>. I.E myapp-dev"
+}
+
+variable "github_team" {
+    description = "Your github team"
```

Comment:
I can see an issue here. Team name actually has to be the name as defined in the github API. For example, it seems it's all lowercase and spaces translated to dashes. You can get the proper name if you visit the team page and look at the URL. How do we let the user know what is the correct value?